app/models/etsource/loader.rb

```diff
@@ -46,8 +46,11 @@ def presets
 
     def area_attributes(area_code)
       @area_attributes[area_code] ||= begin
-        area_attr = Atlas::Dataset.find(area_code).to_hash
+        area = Atlas::Dataset.find(area_code)
+        area_attr = area.to_hash
+        area_attr[:is_derived] = true if area.is_a?(Atlas::Dataset::DerivedDataset)
```

Comment:
It is not (yet) used in other places. I am fine if you change it because the code looks ugly now :smile: